setup/setup_locations_template.sh

```diff
@@ -0,0 +1,2 @@
+echo 'Copy this file to setup/setup_locations.sh and modify the directory to point to the directory of your OpSim output'
```

Comment:
Yes. This is correct(I was running it in that mode for a while but got tired of providing the argument everytime, and did this instead.)